pdns/dnsdistdist/docs/install.rst

```diff
@@ -37,25 +37,25 @@ For Red Hat, CentOS and its derivatives, dnsdist is available in `EPEL <https://
 FreeBSD
 ~~~~~~~
 
-dnsdist is also available in `FreeBSD ports <http://www.freshports.org/dns/dnsdist/>`_.
+dnsdist is also available in `FreeBSD ports <https://www.freshports.org/dns/dnsdist/>`_.
 
 Installing from Source
 ----------------------
 
 In order to compile dnsdist, a modern compiler with C++ 2017 support and GNU make are required.
 dnsdist depends on the following libraries:
 
-* `Boost <http://boost.org/>`_
-* `Lua <http://www.lua.org/>`_ 5.1+ or `LuaJit <http://luajit.org/>`_
-* `Editline (libedit) <http://thrysoee.dk/editline/>`_
+* `Boost <https://boost.org/>`_
+* `Lua <https://www.lua.org/>`_ 5.1+ or `LuaJit <https://luajit.org/>`_
```

Comment:
As noted earlier, this site appears both with and without `www.` and I'm not harmonizing here.